src/documentation/sdks/python-pipeline-dependencies.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: "Managing Python Pipeline Dependencies"
+permalink: /documentation/sdks/python-pipeline-dependencies/
+---
+# Managing Python Pipeline Dependencies
+
+> **Note:** This page is only applicable to runners that do remote execution.
+
+When you run your pipeline locally, the packages that your pipeline depends on are available because they are installed on your local machine. However, when you want to run your pipeline remotely, you must make sure these dependencies are available on the remote machines. This tutorial shows you how to make your dependencies available to the remote workers. Each section below refers to a different source that your package may have been installed from.
+
+**Note:** Remote workers used for pipeline execution typically have a standard Python 2.7 distribution installation. If your code relies only on standard Python packages, then you probably don't need to do anything on this page.
+
+## <a name="pypi"></a>PyPI Dependencies
+
+If your pipeline uses public packages from the [Python Package Index](https://pypi.python.org/pypi), make these packages available remotely by performing the following steps:
+
+**Note:** If your PyPI package depends on a non-Python package (e.g. a package that requires installation on Linux using the `apt-get install` command), see the [PyPI Dependencies with Non-Python Dependencies](#pypiwithnonpython) section instead.
+
+1. Find out which packages are installed on your machine. Run the following command:
+
+        pip freeze > requirements.txt
+
+    This command creates a `requirements.txt` file that lists all packages that are installed on your machine, regardless of where they were installed from.
+
+2. Edit the `requirements.txt` file and leave only the packages that were installed from PyPI and are used in the workflow source. Delete all packages that are not relevant to your code.
+
+3. Run your pipeline with the following command-line option:
+
+        --requirements_file requirements.txt
+
+    The runner will use the `requirements.txt` file to install your additional dependencies onto the remote workers.
+
+**Important:** Remote workers will install all packages listed in the `requirements.txt` file. Because of this, it's very important that you delete non-PyPI packages from the `requirements.txt` file, as stated in step 2. If you don't remove non-PyPI packages, the remote workers attempt to install packages from sources that are unknown to them, and an error occurs.
```

Comment:
The final part of the sentence is confusing a little bit:
`the remote workers attempt to install packages from sources that are unknown to them, and an error occurs`
maybe ?
`The remote workers will fail while attempting to install packages from sources unknown to them.`